scout/server/blueprints/clinvar/templates/clinvar/clinvar_howto.html

```diff
@@ -0,0 +1,44 @@
+{% macro clinvar_howto_modal() %}
+
+<div class="modal fade" id="howto" aria-hidden="true" tabindex="-1">
```

Comment:
These instructions will need to change when I remove the validation buttons, but I didn't want to open that other PR just yet because of code conflicts, but it's coming. Thanks for the review!